torch/_jit_internal.py

```diff
@@ -762,12 +768,21 @@ def should_drop(fn) -> bool:
     attr = get_torchscript_modifier(fn)
     if attr is None:
         return False
-    return attr is FunctionModifiers.UNUSED
+    return attr is FunctionModifiers.UNUSED or attr is FunctionModifiers._DROP
 
 
 def is_ignored_fn(fn) -> bool:
     mod = get_torchscript_modifier(fn)
-    return mod is FunctionModifiers.UNUSED or mod is FunctionModifiers.IGNORE
+    return (
+        mod is FunctionModifiers.UNUSED
+        or mod is FunctionModifiers.IGNORE
+        or mod is FunctionModifiers._DROP
```

Comment:
also here, I think it's safe to leave as it was before (although less confident of this one, check tests to be sure)

AFAICT, is_ignored_fn is mostly used in module scripting. _DROP is probably only going to be used for non-module classes (because modules already drop methods by default, except for the forward method. And supporting this on the forward method might be a bit extra work, because a lot of things assume that modules have a forward method). So, I suspect we don't need to add this.